templates/patterns/list_with_illustration/list_with_illustration.ui_patterns.yml

```diff
@@ -1,36 +1,22 @@
 list_with_illustration:
   label: "List with illustration"
   description: "A list of items with associated illustrations."
-  variants:
-    vertical_images_landscape:
-      label: "Vertical with images (large)"
-      description: "Vertical list with large images."
-    vertical_images_square:
-      label: "Vertical with images (square)"
-      description: "Vertical list with square images."
-    vertical_icons:
-      label: "Vertical with icons"
-      description: "Vertical list with icons."
-    horizontal_images_landscape:
-      label: "Horizontal with images (large)"
-      description: "Horizontal list with large images."
-    horizontal_images_square:
-      label: "Horizontal with images (square)"
-      description: "Horizontal list with square images."
-    horizontal_icons:
-      label: "Horizontal with icons"
-      description: "Horizontal list with icons."
   fields:
     column:
       type: "numeric"
       label: "Column number"
-      description: "Number of columns (only applies to horizontal variants). Maximum of 4 columns."
-      preview: 2
+      description: "Number of columns. Maximum of 4 columns."
+      preview: 1
       escape: false
     zebra:
       type: "boolean"
       label: "Zebra"
-      description: "Use zebra background (only applies to vertical variants)."
+      description: "Use zebra background."
+      preview: true
+    square_image:
```

Comment:
By putting it here instead of doing it in the items themselves we are separating ourselves from the ecl implementation. This is not to say its wrong but it is a choice we must do consciously. @brummbar could you weigh in in this matter?

Reply:
I see not much value in customising the format for each entry. I believe we should control the image style from outside the items.